docs/source/benchmarks.rst

```diff
@@ -135,23 +135,20 @@ Timing benchmarks
 - ``min_run_count``: the function is run at least this many times during
   benchmark. Default: 2
 
-- ``timer``: The timing function to use, which can be any source of
+- ``timer``:
+  ..versionchanged:: 0.4
```

Comment:
Rst directives need blank lines.

Reply:
Thanks, ended up moving this down before the paragraph explaining the change, as that presentation is a bit cleaner